rclpy/src/rclpy/2

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
This file has a strange name `2`. Was it accidentally committed?

Reply:
I'm guessing I accidentally typed `:w 2` at some point. I'll fix it.